cccatalog-api/cccatalog/api/admin.py

```diff
@@ -1,3 +1,26 @@
 from django.contrib import admin
+from cccatalog.api.models import ImageReport, MatureImage
 
-# Register your models here.
+
+@admin.register(ImageReport)
+class ImageReportAdmin(admin.ModelAdmin):
+    list_display = ('reason', 'status', 'image_url', 'description')
+    list_filter = ('status', 'reason')
+    list_display_links = ('status',)
+    actions = None
```

Comment:
Can you remove this unused variable?

Reply:
It's an internal variable used by Django. It hides the default "delete all selected reports" action, which doesn't fit into this flow (we want to preserve reports).

Reply:
Ah, right